contrib/lib/src/databases.rs

```diff
@@ -203,6 +210,21 @@
 //! # }
 //! ```
 //!
+//! Tieing your application to a different RDBMS is simple, and follows the same pattern.
+//! Here's an example using Postgres with Diesel as the driver:
+//!
+//! ```rust
+//! # extern crate rocket;
+//! # #[macro_use] extern crate rocket_contrib;
+//! # #[cfg(feature = "diesel_postgres_pool")]
+//! # mod test {
+//! use rocket_contrib::databases::diesel;
```

Comment:
Can we use a non-Diesel driver here, just so it's not too similar?

Reply:
Sure, I'll go ahead and make the adjustment

Reply:
@jebrosen just updated with a new commit to use Rust-Postgres as the driver.